localisation/00_wc_random_artifact_names.csv

```diff
@@ -92,6 +92,19 @@ artifact_legendary_origin_8;[GetGrammaticalNumberThisThese] [This.GetName] [GetG
+artifact_looted_origin_2;Before [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] become a war trophy of [target_raiding_ruler.GetTitledFirstName], [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] belonging to a family living in [target_looted_holding.GetName], that ceased to exist after the encounter with unidentified armed forces.;;;;;;;;;;;;;x
+artifact_looted_origin_3;By the right of might, [target_raiding_ruler.GetTitledFirstName] become the lawful owner of [GetGrammaticalNumberThisThese] [This.GetName] after [target_raiding_ruler.GetHerHis] army sacked [target_looted_holding.GetName].;;;;;;;;;;;;;x
+artifact_looted_origin_4;Once upon a time, in [target_looted_holding.GetName] lived a man who thought that resistance to evil by force is morally wrong. Unfortunately for him, troops of [target_raiding_ruler.GetTitledFirstName] did not care about that when they took [GetGrammaticalNumberThisThese] [This.GetName] from his dead body.;;;;;;;;;;;;;x
+artifact_looted_origin_5;A poor widow that lived in [target_looted_holding.GetName] did not sell [GetGrammaticalNumberThisThese] [This.GetName] in order to get some food only because [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] the last memento of her husband. When raiders of [target_raiding_ruler.GetTitledFirstName] come, they send that widow to her husband.;;;;;;;;;;;;;x
+artifact_looted_origin_6;Some people tell that subordinates of [target_raiding_ruler.GetTitledFirstName], used "the supreme authority from which all other authorities are derived" to get [GetGrammaticalNumberThisThese] [This.GetName] when they were in [target_looted_holding.GetName]. Other people tell that these subordinates just used violence.;;;;;;;;;;;;;x
+artifact_looted_origin_7;Population of [target_looted_holding.GetName] no longer can voice their complaints about expropriation of [GetGrammaticalNumberThisThese] [This.GetName] in favour of [target_raiding_ruler.GetTitledFirstName], even if they wanted to.;;;;;;;;;;;;;x
+artifact_looted_origin_8;Looters of [target_raiding_ruler.GetTitledFirstName] found [GetGrammaticalNumberThisThese] [This.GetName] when they pillaged [target_looted_holding.GetName].;;;;;;;;;;;;;x
+artifact_looted_origin_9;Caravan that was resting in [target_looted_holding.GetName] carried [GetGrammaticalNumberThisThese] [This.GetName] among their cargo. Due to orders of [target_raiding_ruler.GetTitledFirstName], caravaneers were put to eternal rest and artifact[GetGrammaticalNumberEndingPluralS] [GetGrammaticalNumberWasWere] put to [target_raiding_ruler.GetHerHis] treasury.;;;;;;;;;;;;;x
+artifact_looted_origin_10;Initially [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] gifted to inhabitants of [target_looted_holding.GetName] by Higher Power, later [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] presented to [target_raiding_ruler.GetTitledFirstName] by [target_raiding_ruler.GetHerHis] raiders.;;;;;;;;;;;;;x
```

Comment:
Made a couple changes.

Not sure about the Higher Power part. Is it referring to a deity, or simply a higher authority? If the latter, it doesn't need to be capitalized, if the former, I would change it to be more clear.
```suggestion
artifact_looted_origin_10;Initially [GetGrammaticalNumberThisThese] [This.GetName] [GetGrammaticalNumberWasWere] gifted to the inhabitants of [target_looted_holding.GetName] by a Higher Power. Later, however, [GetGrammaticalNumberItThey] [GetGrammaticalNumberWasWere] presented to [target_raiding_ruler.GetTitledFirstName] by [target_raiding_ruler.GetHerHis] raiders.;;;;;;;;;;;;;x
```

Reply:
> Is it referring to a deity, or simply a higher authority?

To vague cosmic power, so, first is more close.